Automattic/wp-calypso
# E2e tests for VideoPress Onboarding Tailored Flow

#### Proposed Changes

* Adds End-to-end tests for the VideoPress Onboarding Tailored flow.

#### Testing Instructions



* See instructions in README.md under tests/e2e on how to set up.
* Run `yarn jest specs/onboarding/setup__videopress.ts`
* Tests should pass.



Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->



Related to #
